bin/inference/pycbc_inference_plot_samples

```diff
@@ -23,18 +23,26 @@ import logging
 from matplotlib import use
 use('agg')
 from matplotlib import pyplot as plt
+from matplotlib import rc
+import numpy
 import pycbc
 from pycbc import (results, transforms)
 from pycbc import __version__
 from pycbc.inference import (option_utils, io)
 import sys
 
 # command line usage
-parser = argparse.parser = io.ResultsArgumentParser()
+parser = argparse.parser = io.ResultsArgumentParser(skip_args=['walkers'])
 parser.add_argument("--verbose", action="store_true", default=False,
                     help="Print logging info.")
 parser.add_argument("--version", action="version", version=__version__,
                     help="show version number and exit")
+parser.add_argument("--walkers", nargs='+', default=None,
+                    help="Walker indices from the temperature `--temps`  to "
+                         "plot. Options are 'all' or one or more walker "
+                         "indices. Default is to plot the average of all "
+                         "walkers from the common temperature `--temps` for "
+                         "the input `--parameters`.")
```

Comment:
This is a bit confusing. I'd drop mention of temperatures, especially since they wouldn't make sense for single temperature samplers like `emcee`.